charts/pega/README.md

```diff
@@ -658,6 +658,21 @@ If you are planning to use Cassandra (usually as a part of Pega Customer Decisio
 
 To use an existing Cassandra deployment, set `cassandra.enabled` to `false` and configure the `dds` section to reference your deployment.
 
+The following parameters are available for configuration of the connection to the external Cassandra cluster.
+
+Parameter     | Description                                                                                                                                                                                                                                                                                                                   | Default value
+---           |-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------| ---
+`externalNodes`    | A comma separated list of hosts in the Cassandra cluster. | Empty
+`port` | TCP Port to connect to cassandra. | 9042
+`username` | The username for authentication with the Cassandra cluster. | dnode_ext
+`password` | The password for authentication with the Cassandra cluster. | dnode_ext
+`clientEncryption` | Whether to enable client encryption on the Cassandra connection. | false
```

Comment:
Enable (true) or disable (false) client encryption on the Cassandra connection.

Reply:
I like it and will make the change